### PR TITLE
Add hooks

### PR DESCRIPTION
**Pull request by CarlosE-Feliz**
# Math Magicians: Hooks  🎉 

> Set up hooks to develop a React application💪🏽 

Additional description of the project and its features. ➕ 

## Technical set up

- [x]  Refactor the Calculator component (components/Calculator.js) from a class based component to a functional component with hooks.

- [x]  After the refactor is done, test the app in the browser. It should keep the same functionality.

## Built With ❤️ 
- CSS
- JavaScript
- HTML
- React

### NOTE TO REVIEWER 😉 ❤️  
[Demo](https://xjokeer.github.io/Math-magicians/)

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,68 +1,59 @@
...
-  constructor(props) {
-    super(props);
-    this.state = {
-      total: 0,
-      next: null,
-      operation: null,
-    };
-    this.handleEvent = this.handleEvent.bind(this);
-  }
+function Calculator() {
```
Comment (vichuge) [inline src/components/Calculator.js]: - [x] Please, remember to use ES6 best practices for this. 🐶

**Review by vichuge — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 
You used proper GitHub Flow ✔️
Your linter checks are all passing ✔️
Your PR messages are descriptive ✔️

## Aspects to improve ♻️

- [x] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (CarlosE-Feliz) [reply src/components/Calculator.js]: Hey there, could you please explain to me exactly what do you mean?😥

Comment (CarlosE-Feliz) [reply src/components/Calculator.js]: Do you mean to instead of use a normal function to use an arrow function?

Comment (vichuge) [reply src/components/Calculator.js]: Yes, I was talking about it. 😄

**Review by vichuge — APPROVED** [review]
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

InbornCandidHornet-size_restricted

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (CarlosE-Feliz) [reply src/components/Calculator.js]: Thanks for the feedbacks!

**Pull request merged**